mitodl/open-discussions
# small refactor to CommentForms to allow using WYSIWYG editor to edit posts

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #1097 

#### What's this PR do?

this refactors, slightly, the `CommentForms` setup we have so that we can use the `<Editor />` component for editing posts but not for comments.

#### How should this be manually tested?

Confirm that, if you edit a text post, you get the WYSIWYG editor and it all works as it should.
